Tools/SelectionTool/SelectionTool.cs

```diff
@@ -279,7 +279,12 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 
                     // Can't select this object (it might be locked or static)
                     if (directHoveredObject && !directSelectionCandidate)
+                    {
+                        if (directHoveredObject != null)
```

Comment:
It seems like the first if check does the same thing, is this a Unity-overloaded null operator problem?  (And in that case, can we just move this line up to the first if statement?)